packages/contracts-bedrock/src/safe/LivenessModule2.sol

```diff
@@ -146,8 +148,15 @@ contract LivenessModule2 is ISemver {
         _cancelChallenge(msg.sender);
 
         emit ModuleConfigured(msg.sender, _config.livenessResponsePeriod, _config.fallbackOwner);
+
+        // Verify that any other extensions which are enabled on the Safe are configured correctly.
+        _checkCombinedConfig(Safe(payable(msg.sender)));
     }
 
+    /// @notice Internal helper function which can be overriden in a child contract to check if the guard's
+    ///         configuration is valid in the context of other extensions that are enabled on the Safe.
+    function _checkCombinedConfig(Safe _safe) internal view virtual { }
```

Comment:
What do you think about just making LivenessModule2 and TimelockGuard abstract contracts, then making this not defined instead of an empty implementation? This way (1) we are always forced to implement this in SaferSafes, and (2) it foces us to update all existing unit tests to use SaferSafes instead of these partial contracts, i.e. the unit tests account for any interactions between the two components